controllers/send-widget.controller.es6

```diff
@@ -123,8 +124,18 @@ export default class SendWidgetController {
           this.memoBlocked = true;
         } else {
           this.memoBlocked = false;
-          
         }
+
+        if (this.destination in knownAccounts && knownAccounts[this.destination].requiredMemoType) {
+          this.memo = true;
+          this.memoType = knownAccounts[this.destination].requiredMemoType;
+          let alert = new Alert({
+            text: 'The payment destination (' + knownAccounts[this.destination].name +') requires you to specify a memo to identify your account.',
+            type: Alert.TYPES.WARNING
+          });
+          this.memoAlertGroup.show(alert);
+        }
+
```

Comment:
I've tested this. We need to remove the alert when destination is changed to non-exchange address (but the existing memo should be left as it is).